frontend/src/test/scala/bloop/util/TestUtil.scala

```diff
@@ -661,4 +661,19 @@ object TestUtil {
       DependencyResolution.Artifact("org.testng", "testng", "7.9.0")
     DependencyResolution.resolve(List(testNG), logger)
   }
+
+  def getMunitDep(logger: Logger) = {
+    // TODO: figure out clean way
```

Comment:
There is clean way to do it would be to use BuildInfo data from sbt. We should be able to add binary version if it's not there yet.